cmd/whisperImporter/chunkWriter/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"compress/gzip"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/gocql/gocql"
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/cmd/whisperImporter"
+	"github.com/raintank/metrictank/idx"
+	"github.com/raintank/metrictank/idx/cassandra"
+	"github.com/raintank/metrictank/mdata"
+	"gopkg.in/raintank/schema.v1"
+)
+
+const table_name_format = `metric_%d`
```

Comment:
instead of redefining this const here, why not use the one in mdata? that seems more robust if we ever want to change it.